Documentation/gettingstarted/ciliumendpointslice.rst

```diff
@@ -0,0 +1,103 @@
+``enableCiliumEndpointSlice`` value to ``true``.
+
+.. parsed-literal::
+
+   helm install cilium |CHART_RELEASE| \\
+     --set enableCiliumEndpointSlice=true
+
+Verify that Cilium agent and operator pods are running.
+
+.. code-block:: shell-session
+
+   $ kubectl get pods -n kube-system -l k8s-app=cilium
+   NAMESPACE     NAME           READY   STATUS    RESTARTS   AGE
+   kube-system   cilium-7tnkq   1/1     Running   0          1d
+
+   $ kubectl get pods -n kube-system -l io.cilium/app=operator
+   NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE
+   kube-system   cilium-operator-855979b54b-w8rsm   1/1     Running   0          1d
+
+Validate that the CiliumEndpointSlice CRD has been registered.
+
+.. code-block:: shell-session
+
+   $ kubectl get crds --all-namespaces
+   NAME                                         CREATED AT
+   [...]
+   ciliumendpointslices.cilium.io               2021-11-05T05:41:28Z
+
+Known Issues and Workarounds
+============================
+
+Potential Upgrade Impact
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When performing an upgrade from a non-CES mode to CES mode, the operator
+will need to process all existing CEPs in the cluster and create CESs for
+them. Depending on the load of the cluster, this could take some time.
+
+During this period, new endpoints might experience longer delay before
+their identities will propagate to remote nodes. If a newly created pod
+is affected by this delay, then it may temporarily fail to establish
+connectivity to remote pods in the cluster.
+
+A mitigation strategy is to upgrade the operator from non-CES to CES before
+upgrading agents on each node. This will let it start creating CESs right
+away, while the agents are still watching CEP resources.
+
+Potential Racing Condition when Identity of An Existing Endpoint Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When there's an identity change for any existing resources, e.g., a pod,
+a deployment or a statefulset, although unlikely, the endpoints that
+undergo this change might experience connection disruption.
+
+Root cause for this potential disruption is that when identity of CEPs
+change, the operator will try to re-group/re-batch them into a different
+set of CESs. This breaks the atomic operation of an UPGRADE into that of
+an DELETE and an ADD. If the agent gets the DELETE (from old CES) first,
+it will remove the corresponding CEP's information from the ipcache,
+resulting in traffic to/from said CEP with an UNKNOWN identity.
+
+In current implementation, Cilium adds a delay (default: 1s) before sending
+out the DELETE event. This should greatly reduce the probability of
+connection disruption in most cases.
+
+Egress Gateway Won't Work with CES
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This is a temporary issue until Github Issue #17669 is fixed. If you
```

Comment:
There is a standard way to reference github issues in the docs, I think `:gh-issue:`17669` should do the trick. Might need to double-check how that renders in the middle of the sentence but you can also grep around for `gh-issue` to understand how we typically reference github issues in other parts of the docs.

Reply:
Done.